php/blocks/class-loader.php

```diff
@@ -207,17 +215,19 @@ public function render_block_template( $block, $attributes ) {
 			$type = array( 'preview', 'block' );
 		}
 
-		if ( 'edit' !== $context ) {
+		if ( ! is_admin() ) {
 			/**
 			 * The block has been added, but its values weren't saved (not even the defaults). This is a phenomenon
-			 * unique to frontend output, as the editor fetches is attributes from the form fields themselves.
+			 * unique to frontend output, as the editor fetches its attributes from the form fields themselves.
 			 */
 			$missing_schema_attributes = array_diff_key( $block['fields'], $attributes );
 			foreach ( $missing_schema_attributes as $attribute_name => $schema ) {
 				if ( isset( $schema['default'] ) ) {
 					$attributes[ $attribute_name ] = $schema['default'];
 				}
 			}
+
+			block_lab_enqueue_styles( $block['name'], 'block' );
```

Comment:
Would this method fit well in the `Loader` class, instead of `php/helpers.php`? Though maybe it'll later be used in other classes, I'm not sure.

Reply:
Great idea – done in 6148de0.